lib/erb_lint/linter_config.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module ERBLint
+  class LinterConfig
+    def initialize(config = {})
+      @config = config.dup.deep_stringify_keys
```

Comment:
`deep_stringify_keys` is a active support method. We need to add it as dependency and require it if we want to use.

Reply:
👍 added